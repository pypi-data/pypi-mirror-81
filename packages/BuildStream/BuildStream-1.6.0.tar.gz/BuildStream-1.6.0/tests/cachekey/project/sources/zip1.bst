kind: import
sources:
- kind: zip
  url: https://example.com/releases/1.4/foo-1.4.5.zip
  ref: 6c9f6f68a131ec6381da82f2bff978083ed7f4f7991d931bfa767b7965ebc94b
