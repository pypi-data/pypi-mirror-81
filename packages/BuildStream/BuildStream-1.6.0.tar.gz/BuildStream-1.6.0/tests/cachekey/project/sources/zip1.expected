5a6ac152f8bd88add55172dca76225c1cf01c821e1037184ff54f06baf7fb028